src/binfhe/unittest/UnitTestFunc.cpp

```diff
@@ -0,0 +1,201 @@
+//==================================================================================
```

Comment:
For unit tests, it is better to use toy parameters. Unit tests are supposed to run fast.